doc/guide/Guide_Git_Development.ipynb

```diff
@@ -8,7 +8,7 @@
     }
    },
    "source": [
-    "# Git and GitHub and CLIMADA\n",
+    "# Development and Git and CLIMADA\n",
```

Comment:
Are the pictures in this guide chosen on purpose @ChrisFairless  ? I find them a bit distracting.

Reply:
they were just for the presentation (it pulled random stock images based on the slide's keywords). i've removed them and made a few extra edits on another branch. i'll do a little pull request to merge them into this branch, probably on monday

Reply:
You could also directly commit to this branch by merging your branch into this one (and only change the files that you have updated). This way we have a single pull request.